### PR TITLE
Add Ctrl+g shortcut info to cycle through search results

### PR DESCRIPTION
**Pull request by vviikk**
Just added a line of text to the new tab page for users to know how to search through the page.

### CODE REVIEW

File: src/static/newtab.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ REPLACE_ME_WITH_THE_CHANGE_LOG_USING_SED
 -   `t`/`T` — open a URL in a new tab (`T` to pre-load current URL).
 -   `gt`/`gT` — go to the next/previous tab.
 -   `d` — close the current tab.
--   `/` — open the find search box.
+-   `/` — open the find search box. Use <kbd>ctrl</kbd> + g/G to cycle through search results.
```
Comment (glacambre) [inline src/static/newtab.md]: Hi. Thank you for opening this PR. We usually use `<C-g>` in order to represent chorded keyboard shortcuts. I think it'd be better to use this representation here too. Could you do that?

Comment (bovine3dom) [reply src/static/newtab.md]: Sorry, glacambre, I already merged it :(

Comment (glacambre) [reply src/static/newtab.md]: Argh, merged before I could comment. I'll do it myself then. Thanks again for this PR. :)

**Comment by vviikk** [comment]
Oh, sorry. Hope all is cool. I'm travelling without much internet access.

**Comment by glacambre** [comment]
No problem :)

**Pull request merged**